[2024-02-06T21:22:25.319-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-3/run_id=manual__2024-02-07T00:22:15.993940+00:00/task_id=create_emr_cluster permission to 509
[2024-02-06T21:22:25.465-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-3/run_id=manual__2024-02-07T00:22:15.993940+00:00/task_id=create_emr_cluster permission to 509
[2024-02-06T21:22:25.521-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: how-desafio-3.create_emr_cluster manual__2024-02-07T00:22:15.993940+00:00 [queued]>
[2024-02-06T21:22:25.562-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: how-desafio-3.create_emr_cluster manual__2024-02-07T00:22:15.993940+00:00 [queued]>
[2024-02-06T21:22:25.563-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2024-02-06T21:22:25.878-0300] {taskinstance.py:1382} INFO - Executing <Task(EmrCreateJobFlowOperator): create_emr_cluster> on 2024-02-07 00:22:15.993940+00:00
[2024-02-06T21:22:25.880-0300] {standard_task_runner.py:57} INFO - Started process 965 to run task
[2024-02-06T21:22:25.885-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'how-desafio-3', 'create_emr_cluster', 'manual__2024-02-07T00:22:15.993940+00:00', '--job-id', '334', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmprs34ejpm']
[2024-02-06T21:22:25.886-0300] {standard_task_runner.py:85} INFO - Job 334: Subtask create_emr_cluster
[2024-02-06T21:22:26.031-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-3/run_id=manual__2024-02-07T00:22:15.993940+00:00/task_id=create_emr_cluster permission to 509
[2024-02-06T21:22:26.034-0300] {task_command.py:416} INFO - Running <TaskInstance: how-desafio-3.create_emr_cluster manual__2024-02-07T00:22:15.993940+00:00 [running]> on host 14111-NB.
[2024-02-06T21:22:26.286-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Filipe' AIRFLOW_CTX_DAG_ID='how-desafio-3' AIRFLOW_CTX_TASK_ID='create_emr_cluster' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:22:15.993940+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-07T00:22:15.993940+00:00'
[2024-02-06T21:22:26.287-0300] {emr.py:742} INFO - Creating job flow using aws_conn_id: aws_default, emr_conn_id: emr_default
[2024-02-06T21:22:26.312-0300] {logging_mixin.py:151} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/providers/amazon/aws/operators/emr.py:750 UserWarning: Unable to find Amazon Elastic MapReduce Connection ID 'emr_default', using an empty initial configuration. If you want to get rid of this warning message please provide a valid `emr_conn_id` or set it to None.
[2024-02-06T21:22:26.330-0300] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-02-06T21:22:26.331-0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-02-06T21:22:27.816-0300] {emr.py:756} INFO - Job flow with id j-1VTDUAHE0XE8X created
[2024-02-06T21:22:28.201-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=how-desafio-3, task_id=create_emr_cluster, execution_date=20240207T002215, start_date=20240207T002225, end_date=20240207T002228
[2024-02-06T21:22:28.273-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-02-06T21:22:28.345-0300] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
